packages/eslint-config-react-app/index.js

```diff
@@ -174,8 +160,8 @@ module.exports = {
     'no-template-curly-in-string': 'warn',
     'no-this-before-super': 'warn',
     'no-throw-literal': 'warn',
-    'no-undef': 'error',
-    'no-restricted-globals': ['error'].concat(restrictedGlobals),
+    'no-undef': 'warn',
```

Comment:
This should intentionally be an error. We want to train people to qualify globals with `window.` when they mean to use a global. It's not a stylistic choice, but to clearly separate the concept of npm dependencies from a concept of the globals.